src/main/resources/application.properties

```diff
@@ -3,14 +3,14 @@ management.endpoints.web.exposure.include=health,metrics,prometheus,liveness
 opentracing.jaeger.log-spans = false
 
 # Server configuration
-server.port = ${SERVICE_PORT:8080}
```

Comment:
We can keep the default port here. If you want to run it on a different port, it is better to expose it rather than changing it here

Reply:
reverted the file